src/js/components/DataChart/DataChart.js

```diff
@@ -0,0 +1,140 @@
+import React, { useMemo } from 'react';
+import { Box } from '../Box';
+import { Chart, calcs } from '../Chart';
+import { Grid } from '../Grid';
+import { Stack } from '../Stack';
+import { Text } from '../Text';
+
+const DataChart = ({
+  chart,
+  data,
+  steps,
+  thickness: thicknessProp,
+  xAxis,
+  yAxis,
+  ...rest
+}) => {
+  const charts = useMemo(() => (Array.isArray(chart) ? chart : [chart]), [
+    chart,
+  ]);
+
+  const values = useMemo(() => {
+    const result = {};
+    charts.forEach(({ key }) => {
+      if (!result[key]) {
+        result[key] = data.map(d => d[key]);
+      }
+    });
+    return result;
+  }, [charts, data]);
+
+  const { axis, bounds, thickness } = useMemo(
+    () =>
+      calcs(values[Object.keys(values)[0]], {
+        steps,
+        thickness: thicknessProp,
+      }),
+    [steps, thicknessProp, values],
+  );
+
+  const xGuide = useMemo(
+    () =>
+      axis[0].map((_, i) => {
+        if (xAxis && xAxis.guide) {
+          if (i === 0) return 'left';
+          if (i === axis[0].length - 1) return 'right';
+          return true;
+        }
+        return undefined;
+      }),
+    [axis, xAxis],
+  );
+
+  const yGuide = useMemo(
+    () =>
+      axis[1].map((_, i) => {
+        if (yAxis && yAxis.guide) {
+          if (i === 0) return 'top';
+          if (i === axis[1].length - 1) return 'bottom';
+        }
+        return undefined;
+      }),
+    [axis, yAxis],
+  );
+
+  /* eslint-disable react/no-array-index-key */
+  return (
+    <Grid
```

Comment:
we should have a disclaimer for IE11 on docs.

Reply:
@ShimiSun Has there been any talk of adding ie11 polyfills?  We are considering our grid options for GLC and need to support ie11.

Reply:
Not yet, it could be a great contribution to the grommet if you guys are already working on it.

Reply:
It should work on IE11 now.